chroot_setup

```diff
@@ -0,0 +1,440 @@
+    PATH='/QOpenSys/pkgs/bin:/QOpenSys/pkgs/sbin:/QOpenSys/usr/bin:/QOpenSys/usr/sbin:'
+    LIBPATH='/QOpenSys/pkgs/lib:/QOpenSys/usr/lib'
+    os.environ['PATH'] = PATH
+    os.environ['LIBPATH'] = LIBPATH
+    print("**********************")
+    print("Live IBM i session (changes made).")
+    print("**********************")
+
+    if (args.verbose):
+      print('PATH=' + PATH)
+      print('LIBPATH=' + LIBPATH)
+
+  else:
+    # CHROOT_DEBUG = True
+    print("**********************")
+    print("Not IBM i, no action is taken (debug flow purpose only).")
+    print("**********************")
+    # exit -1
+
+
+# Install any yum packages passed with the -i flag
+def yum_install(package_list: list):
+  os.environ["LIBPATH"] = '/QOpenSys/pkgs/lib:/QOpenSys/usr/lib'
+  install_template = Template('/QOpenSys/pkgs/bin/yum $yes --installroot=$chroot_dir install $package')
+  auto_yes = '-y' if args.auto_yes else ''
+  for package in package_list:
+    command = install_template.substitute(yes=auto_yes, chroot_dir=args.chroot_directory, package=package)
+    logging.debug(command)
+    os.system(command)
+
+#
+# MAIN
+#
+if __name__ == '__main__':
+  lst = 0
+  ops = 0
+  qsys = 0
+  CHROOT_DIR = ""
+  CHROOT_LIST = ""
+  SCRIPT_DIR = os.path.dirname(os.path.realpath(__file__)) + "/config"
+
+  print(SCRIPT_DIR)
+
+  auto_yes = args.auto_yes
+  initial_check()
+
+  # This is done because the arg may be a relative path name.
+  # Need to be done before any 'cd'
+  TEMP_DIR = os.path.realpath(args.chroot_directory)
+  print(TEMP_DIR)
+
+  # if -v was passed, need to turn logging to DEBUG so all logging messages in the
+  # code are printed
+  if (args.verbose):
+    logging.basicConfig(level=logging.DEBUG)
+
+  logging.debug("Script directory: " + SCRIPT_DIR)
+  os.chdir(SCRIPT_DIR)
+  logging.debug("PWD: " + os.getcwd())
+
+  if not TEMP_DIR.startswith('/QOpenSys/'):
+    print("\nERROR: [CHROOT DIRECTORY] Must begin with /QOpenSys/...\n")
+    parser.print_help
+    sys.exit(-2)
+
+  CHROOT_DIR = TEMP_DIR
+
+  if not os.path.isdir(CHROOT_DIR):
+    print(CHROOT_DIR + " does not exist.")
+    logging.debug(f"Creating directory at {CHROOT_DIR}")
+    os.mkdir(CHROOT_DIR)
+  else:
+    if not auto_yes:
+      print(CHROOT_DIR + " directory already exists")
+      val = input("Would you like to continue chroot setup? [y/N]")
+      if not val == "y":
+        print("Bye")
+        sys.exit(-4)
+
+
+  # Validate CHROOT TYPE Argument(s)
+  if not args.chroot_type:
+    # Only chroot_directory was provided to the script:
+    # Ask if minimal with includes chroot is desired
+    if not auto_yes:
+      val = input("Would you like to create a minimal chroot w/ includes @ " + CHROOT_DIR + "? [y/N]:")
+      if not val == "y":
+        print("Specify your desired [CHROOT TYPE]")
+        parser.print_help
+        sys.exit(-5)
+
+    mylist = []
+    mylist.append(SCRIPT_DIR + "/chroot_includes.lst")
+    mylist.append(SCRIPT_DIR + "/chroot_minimal.lst")
```

Comment:
After #75, we should now use default.lst instead.

```suggestion
mylist.append(SCRIPT_DIR + "/default.lst")
```

Reply:
I will push a commit with this change

Reply:
This branch would need to be rebased with master to pull in default.lst, for testing purposes.